mlperf_logging/compliance_checker/training_5.1.0/closed_llama31_8b.yaml

```diff
@@ -25,7 +25,6 @@
 - KEY:
     NAME:  opt_learning_rate_decay_steps
     REQ:   EXACTLY_ONE
-    CHECK: " v['value'] == 1200000 " 
```

Comment:
Why is this removed? We need to make sure all submissions set it to `1200000` so that the cosine decay matches.

Reply:
Let's do the actual check to ensure that all submissions indeed set this correctly. 

So can you please add this to both llama3.1 8b and llama3.1 405b as well?  
```
- KEY:
    NAME:  opt_learning_rate_decay_steps
    REQ:   EXACTLY_ONE
    CHECK: " v['value'] == math.ceil(1_200_000 / s['global_batch_size'] ) - math.ceil(8000 * 1152 / s['global_batch_size'] ) "
```